connectors/sources/dropbox.py

```diff
@@ -864,8 +866,14 @@ async def get_content(
         Returns:
             dictionary: Content document with _id, _timestamp and attachment content
         """
+        if not doit:
+            self._logger.info("Skipping attachment downloading")
+            return
+
         file_size = int(attachment["size"])
-        if not (doit and file_size > 0):
+
+        if file_size <= 0:
+            self._logger.info(f"Skipping file as file size is {file_size}")
```

Comment:
```suggestion
            self._logger.warn(f"Skipping file '{attachment["name"]}' as file size is {file_size}")
```
Or, should we raise? This feels like a bug if size is negative.

Reply:
I think `warn` level is enough because something went wrong with a a single attachment.